test/run-pass/language-demonstration/rnp-and-classic-notation/test.yaml

```diff
@@ -0,0 +1,12 @@
+- compile:
+    - check-classic-notation.ieo
+    - classic-notation-to-rnp.ieo
+    - rnp-to-classic-notation.ieo
+    - main.ieo
+    - utility.ieo
```

Comment:
until #34 is implemented, the order of modules is important; the first module must be the one that contains `main()` function